plugins/base/src/main/kotlin/translators/descriptors/DefaultDescriptorToDocumentableTranslator.kt

```diff
@@ -499,12 +499,15 @@ private class DokkaDescriptorVisitor(
             .filter { it.kind == ClassKind.ENUM_ENTRY }
             .map { enumEntryDescriptor(it, parent) }
 
-
-    private fun DeclarationDescriptor.resolveDescriptorData(platformData: PlatformData?): PlatformDependent<DocumentationNode> =
-        if (platformData != null) PlatformDependent.from(
-            platformData,
-            getDocumentation()
-        ) else PlatformDependent.expectFrom(getDocumentation())
+    private fun DeclarationDescriptor.resolveDescriptorData(platformData: PlatformData?): PlatformDependent<DocumentationNode> {
+        val documentation = getDocumentation()
```

Comment:
`getDocumentation().let {}`, no need for this temp variable there IMHO

Reply:
Done